sql/mysqld.cc

```diff
@@ -6108,6 +6108,7 @@ static int init_server_components() {
 #endif
     locked_in_memory = 0;
 
+  mysql_bin_log.print_last_acked_pos();
```

Comment:
It seems like this function is only called during server initialization, vs. whenever a semi-sync slave connects to the server?

It seems like ReplSemiSyncMaster::add_slave() would be a better place for this call?